core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -161,8 +161,11 @@
       width: 36px;
       z-index: 1;
     }
-    .conversation-skin-oppia-avatar {
-      left: -20px;
+    .conversation-skin-supplemental-card-container, .conversation-skin-oppia-avatar {
```

Comment:
Yes, saw the comma practise at all CSS properties in the file. Hence added it too.